src/coverlet.msbuild.tasks/MSBuildLogger.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using ILogger = Coverlet.Core.Logging.ILogger;
+
+namespace Coverlet.MSbuild.Tasks
+{
+    class MSBuildLogger : ILogger
+    {
+        private readonly TaskLoggingHelper _log;
+
+        public MSBuildLogger(TaskLoggingHelper log) => _log = log;
+
+        public void LogVerbose(string message) => _log.LogMessageFromText(message, MessageImportance.Low);
```

Comment:
`LogMessage(message,MessageImportance.Low)` ?

Reply:
If the log level is verbose, MS Build logging equivalence is MessageImportance.Low.

Reply:
Yes I mean that we should use `LogMessage` and not `LogMessageFromText` guide says that is used for error and warning and this is verbose info, right? `Logs an error message or warning from the given line of text.` https://docs.microsoft.com/en-us/dotnet/api/microsoft.build.utilities.tasklogginghelper.logmessagefromtext?redirectedfrom=MSDN&view=netframework-4.7.2#Microsoft_Build_Utilities_TaskLoggingHelper_LogMessageFromText_System_String_Microsoft_Build_Framework_MessageImportance_

Reply:
Oh, I thought you were wondering why it was that.  Didn't notice you were suggesting to use LogMessage instead of LogMessageFromText